spec/serializers/v2/move_serializer_spec.rb

```diff
@@ -60,15 +60,14 @@
         supplier: create(:supplier),
       )
     end
-    let!(:event) { create(:event_move_cancel, eventable: move) }
 
     let(:adapter_options) { { include: described_class::SUPPORTED_RELATIONSHIPS } }
 
     before { ActiveStorage::Current.host = 'http://www.example.com' } # This is used in the serializer
 
     it 'contains all included relationships' do
       expect(result[:included].map { |r| r[:type] })
-        .to match_array(%w[people ethnicities genders locations locations profiles moves documents prison_transfer_reasons court_hearings suppliers events moves])
+        .to match_array(%w[people ethnicities genders locations locations profiles moves documents prison_transfer_reasons court_hearings suppliers])
```

Comment:
not sure why the extra event was removed?

Reply:
The `timeline_events` stuff which returns `events` is already covered extensively a little lower down in this same file so no sense testing it again here.